src/duqtools/merge.py

```diff
@@ -11,34 +12,47 @@
 info, debug = logger.info, logger.debug
 
 
-def merge(*, merge_all: bool, **kwargs):
+def merge(*, merge_all: bool, target: str, template: str, handles: List[str],
+          runfile: Optional[str], variables: Optional[List[str]], **kwargs):
     """Merge as many data as possible."""
-    template = ImasHandle.parse_obj(cfg.merge.template)
-    target = ImasHandle.parse_obj(cfg.merge.output)
+    template = ImasHandle.from_string(template)
+    target = ImasHandle.from_string(target)
 
-    handles = read_imas_handles_from_file(cfg.merge.data).values()
+    handles = [ImasHandle.from_string(handle) for handle in handles]
+    if runfile:
+        handles = handles + list(read_imas_handles_from_file(runfile).values())
+
+    handles = set(handles)  # Remove duplicate handles
 
     for handle in handles:
         op_queue.add(action=lambda: None,
```

Comment:
We can use `op_queue.add_no_op(...)` here. I changed a bunch of these in `main` just now, but maybe that still needs to be merged in.

Reply:
The default text color of `add_no_op` is red, and I wanted green, I can add the click format to green here ...

Reply:
Alright, makes sense 👍